src/tracing/index.ts

```diff
@@ -17,24 +17,20 @@ const AMPLITUDE_DUMMY_KEY = '00000000000000000000000000000000'
 export const STATSIG_DUMMY_KEY = 'client-0000000000000000000000000000000000000000000'
 
 Sentry.init({
+  // general configuration:
   dsn: process.env.REACT_APP_SENTRY_DSN,
   release: process.env.REACT_APP_GIT_COMMIT_HASH,
   environment: getEnvName(),
+  // exception reporting:
   enabled: isSentryEnabled(),
+  // performance tracing:
```

Comment:
nit: capital letter

Reply:
I would drop this comment (no added value) or - alternatively - provide additional context what is sample rate and why we're setting it to this one. 

Also, are we satisfied with `0`? What does that mean in practice? Shouldn't we:
- keep it undefined
- throw an error when env variable not provided (to make sure we don't run into issues in production when one forgets to set the value - in dev, it will most likely exist anyway due to predefined `.env` file)

Reply:
In this case, `0` is functionally equivalent to `undefined`, and it cannot be guaranteed to exist in dev mode so I think its best to not throw (unless we decide to also define it in `.env`, but I think that's dangerous because then removing it in `.env.production` would cause us to use the dev value).